src/i18n/strings.ftl

```diff
@@ -256,8 +256,8 @@ DISCARD-CHANGES = DISCARD CHANGES
 DISCARD-OBSERVATION = DISCARD OBSERVATION
 
 DISCARD-X-OBSERVATIONS = {$count ->
-    [one] DISCARD OBSERVATION
-    *[other] DISCARD {$count} OBSERVATIONS
+[one] DISCARD OBSERVATION
+*[other] DISCARD {$count} OBSERVATIONS
```

Comment:
Are these manual? I don't think we have linting set up for the ftl files, and all the examples at https://projectfluent.org/ use indentation so I don't think this is conventional. If this is something your editor was doing automatically could you turn it off?